frontend/src/modules/dashboard/components/ValueInfo.tsx

```diff
@@ -1,17 +1,22 @@
 /**
- * @summary Re-usable UI wrapper for displaying Large size `ValueInfo`
+ * @deprecated
+ * @summary Re-usable UI wrapper for displaying Value Info
+ *
+ * @file ValueInfo is a configurable component on Layout level
+ * Each Value Info must have 1 main Container & optional 2 subContainer
+ * This matches with Dashboard  Display Value Layout design
  *
  */
-import { Grid, makeStyles, Typography } from '@material-ui/core';
+import { Grid, makeStyles, Theme, Typography } from '@material-ui/core';
 import React, { useEffect, useState } from 'react';
 import { shallowEqual, useSelector } from 'react-redux';
-import { getAlarmLimitsCurrent } from '../../../store/controller/selectors';
+import { getAlarmLimitsRequest } from '../../../store/controller/selectors';
```

Comment:
So this is the misleading part i described in PR description, since I restructured `ValueInfo` and moved it to components, it thinks I change `LargeValueInfo`, but `ValueInfo` always had `requested` alarm limits see [ValueInfo L13](https://github.com/pez-globo/pufferfish-software/blob/develop/frontend/src/modules/dashboard/containers/ValueInfo.tsx#L13), this was done because when we are passing AlarmLimits from `HFNCControl` in multistep popup, initial values are not set, so we show the requested one's until they are set to some value we care. 
whereas on the Dashboard we still pass the current values through the prop [HfncMainView](https://github.com/pez-globo/pufferfish-software/pull/407/files#diff-14f0ce1c307005375c5d869583340a77eaa8a39d29141c2e3aef874dd8bad816R64)

Reply:
@ethanjli so could you just take a look at HFNCControl in multi step popup, now it's similar to components on dashboard, I did not change that yet (go back to original padding) i feel we should? Let me know your thoughts.